docs/blog-guidelines.md

```diff
@@ -0,0 +1,54 @@
+
+The content on our blog consists of:
+* Use cases and success stories
+* Technical content and how-to’s
+* Stories, updates and opinions about JSON Schema 
+* Articles to present JSON Schema tooling
+* Industry insight into JSON Schema adoption
+* JSON Schema Community updates and reports
+
+Other topics are welcome, but it needs to have a direct link to the JSON Schema Community.
+
+We accept content from JSON Schema maintainers and contributors, implementation maintainers and JSON Schema Champions. 
+
+Blog type definitions:
+* Tooling blog: Any blog post on a JSON Schema implementation. 
+* Community blog: Any blog post about the Community written by a JSON Schema maintainer or contributor.
+* Ambassador blog: This is a blog written by one of the JSON Schema Ambassadors.
+* Opinion: This is a blog written by one of the JSON Schema maintainers or contributors to share a JSON Schema related opinion.
+* Update: This is a blog written by one of the JSON Schema maintainers to update about the project evolution.
+* Engineering: Any JSON Schema related technical post.
+
+## Content ##
+The JSON Schema blog audience is developers, IT operators, and JSON Schema enthusiasts. Keep this in mind as you develop your blog content.
+
+Some things to think about:
+* Posts from members must be vendor-neutral. The post may mention a vendor’s name as it relates to specific open source projects, project deployments, adoption paths, their hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the Community, but it shouldn’t feel like an advertisement for your services or company. We do not accept announcements or press releases to the JSON Schema blog.
+* The most interesting posts are how-to blogs, blogs with technical content, and those that teach or show how to do something in a way others may not have thought of.
+* Blog posts that show hurdles that were encountered and explain how they were overcome often do very well as the Community is looking to learn.
+* Critical commentary or broad issues must be approached with sensitivity, professionalism, and tact in a way that is beneficial and positive for the Community.
+* Your post must be your content, but can be published elsewhere with a right to republish. All content should have an author and be published Creative Commons with Attribution.
+
+Members can add 1-2 sentences at the end of their blog with a link to an external site such as their website or a gated asset. Examples for best practices:
+* Share contact info of  speaker - twitter, slack, email
+* Links to company websites
+* Links to projects websites or Github
+* Can advertise if they are hiring and who to contact if interested
+
+Note: This is not a link to a JSON Schema website or form on the JSON Schema website.
```

Comment:
```suggestion
Note: This is not a link to a JSON Schema website or from on the JSON Schema website.
```

Can you help me understand this sentence please? I'm not sure what it means 😅

Reply:
Honestly I don't know what it means. I saw it in the blog guidelines I took as reference and kept it, but I think will be better to remove it.